docs.rb

```diff
@@ -12,24 +12,17 @@ def self.plugins
 
 # gets readme files in the folders
 def self.get_readmes
+	c = File.open("docs.md", "w")
 	get_folders = plugins
-	# readmes = []
 	get_folders.each do |folder|
 		Dir.glob('./' + folder + '/*.md') do |md|
```

Comment:
Is this gonna take any .md file in the directory and output them as-is, right?

Also, didn't we say we were to handle asciidoc by default/as a first step?